src/featureSelectors.ts

```diff
@@ -76,12 +87,12 @@ export const makeGetDenormalizedData = ({
   name: string;
   entity: string;
 }) => {
-  const getHasFeatureFetchedSuccess = makeGetHasFeatureFetchedSuccess({ name });
+  const getIsSliceSuccessful = makeGetIsSliceSuccessful({ name });
   return createSelector(
     getFeatureSlices,
-    getHasFeatureFetchedSuccess,
-    (slices: FeatureSlices<FeatureSlice>, hasFetchedSuccessfully: boolean) => {
-      if (!hasFetchedSuccessfully || !slices.entities) {
+    getIsSliceSuccessful,
+    (slices: FeatureSlices<FeatureSlice>, getIsSliceSuccessful: boolean) => {
+      if (!getIsSliceSuccessful || !slices.entities) {
         return {};
```

Comment:
maybe this should return undefined? Not sure why I was returning an empty object here